src/Components/SigDetailsTable/NewSigDetailsTable.js

```diff
@@ -152,6 +157,29 @@ const NewSigDetailsTable = ({ ruleName, affectedCount, isEmptyAccount }) => {
     fetchDetailsTableGroups(true);
   }, [tableVars]);
 
+  useEffect(() => {
+    stateSet({
+      type: 'setTableVars',
+      payload: {
+        displayName: searchParams.get('displayName') || '',
+        offset: 0,
+      },
+    });
+
+    stateSet({
+      type: 'setTableVars',
+      payload: {
+        sigMatchStatusFilter: searchParams.getAll('status').length
+          ? searchParams.getAll('status')
+          : undefined,
+        hostGroupFilter: searchParams.getAll('hostWorkspace').length
+          ? searchParams.getAll('hostWorkspace')
+          : undefined,
+      },
+      offset: 0,
+    });
```

Comment:
Same as with the state updates below. It looks to me that is should be possible to have only one state change triggered here that updates all three filters at once.

Reply:
It also looks to me that this search params filters feature can be consolidated into a hook that can be used in both tables.